### ISSUE TITLE
VLC video source skips the first and last second from mp3 files in a playlist

### ISSUE THREAD

**Issue by COOLIGUAY** [obsproject/obs-studio#9345]
### Operating System Info

Windows 10

### OBS Studio Version

29.1.3

### OBS Studio Log URL

https://obsproject.com/logs/AHeDnIdLZ0E3nwTS

### Expected Behavior

Plays a playlist of mp3 files with the vlc video source without skipping the first and last second of each file.

### Current Behavior

When playing a list of mp3 files with the vlc video source, it skips the first and last second (aprox.) of each mp3.

### Steps to Reproduce

1. Add an vlc video source with some mp3 files that start immediately and do not have one or more seconds of silence before starting
2. Each mp3 file skips the first and the last second

### Anything else we should know?

I have tried with several versions of VLC player 64bit with the same result (2.2.8, 3.0.0, 3.0.10, 3.0.16 and 3.0.17.4).

Tested in another pc with a similar config (i7 6700, GTX980, WIN10).

**Comment by gxalpha** [comment]
Does this happen in audio monitoring only, or is it also present in the actual recording file?

**Comment by COOLIGUAY** [comment]
Thank you for your time!

I just checked what you ask.

With Monitor Off, no audio on the monitor as expected.
In the recording, the first second it's ok but the last second of each mp3 file is skipped.

With Monitor Only, the first second is skipped on the monitor but the last is ok*.
In the recording, as expected, no audio.
If you add a global audio device (taking that audio), in the recording the first and last second are skipped.

With Monitor and Output, on the monitor it happens the same as with Monitor Only, first second is skipped.
In the recording, like Monitor Off, the first second it's ok but the end is skipped.

*by saying "ok", it might cut a few milliseconds, but it's not really important/annoying.

**Comment by yongsongTang** [comment]
I also encountered a similar problem, I would like to know your plans on this issue, because this issue is important to me.@gxalpha

**Comment by XcantloadX** [comment]
Any solution or work around to this?

**Comment by axwax** [comment]
My workaround is to use VLC standalone with the "[Now-Playing-in-Texts](https://github.com/un-pogaz/Now-Playing-in-Texts)" plugin to pass on metadata. This solution is far from perfect, but better than having the first second of a track cut off.

Just a heads-up for anyone stumbling across this issue - this bug is still happening in OBS 30.2.3

**Comment by Jimmysaaan** [comment]
Here's what I'm experiencing:

https://github.com/user-attachments/assets/cf668997-79ba-4492-a9b0-9245f90903ff

(the first clip is from [here](https://www.sample-videos.com/video321/mp4/720/big_buck_bunny_720p_2mb.mp4) and the second is from [here](https://www.sample-videos.com/video321/mp4/720/big_buck_bunny_720p_1mb.mp4))

I have `VLC Video Source` set to monitor and output.
OBS version: 31.1.2
VLC version: 3.0.20

This issue is probably the same as #3712 (which is a duplicate of #7552), which #8147 claims to have fixed. People are also complaining about the issue on the [forum](https://obsproject.com/forum/threads/vlc-video-source-audio-cuts-off-early.114866/). Is #8147 implemented in the latest releases?
